### DISCUSSION TITLE
Please provide an action keyword to search for folders only

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1174] [Ideas]
Sometimes we are looking for a folder that contains multiple files on a given subject.
At that time, we are not looking for files that have the same search pattern in their names.

FL should have a (user-definable) **action keyword** that forces only a folder search.

**Comment by taooceros** [comment]
https://github.com/Flow-Launcher/Flow.Launcher/issues/769
